cli/src/declarations.ts

```diff
@@ -453,6 +453,16 @@ export interface CapacitorConfig {
      * @default true
      */
     handleApplicationNotifications?: boolean;
+
+    /**
+     * On iOS 16.4 and greater, enable debuggable web content for release builds.
```

Comment:
maybe also mention that it requires Xcode 14.3?

Reply:
Yep - I'll add that now.